mne_connectivity/spectral/epochs.py

```diff
@@ -830,8 +831,8 @@ def spectral_connectivity_epochs(data, names=None, method='coh', indices=None,
         Use adaptive weights to combine the tapered spectra into PSD.
         Only used in 'multitaper' mode.
     mt_low_bias : bool
-        Only use tapers with more than 90%% spectral concentration within
-        bandwidth. Only used in 'multitaper' mode.
+        Only use tapers with more than 90 percent spectral concentration within bandwidth.
```

Comment:
@larsoner @sam-s  not sure why, but the '%%' throws an error using verbose(?). Also if I replace it w/ '\%', it throws an error as well. Not worth the trouble to figure out I guess.

Reply:
You probably need `%%%%`. This is because both `@fill_doc` and `@verbose` will try to complete these, so the first time through you need `%%%%` so this reduces to `%%`, which on the second `%`-operation reduces to just `%`. Or to put it in code:
```
>>> '%%%%' % () % ()
'%'
```